recipes/python-liquid/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "python-liquid" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/python-liquid-{{ version }}.tar.gz
+  sha256: cc9a9fdfb413a70a8891b9fa48c07e9584cc4324554798b0227527050d2681c0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - markupsafe >=2.0.0
```

Comment:
Since this is an option, you can also specify it as run_constrained. Or is this normally used?

Reply:
`markupsafe` has been moved to `run_constrained`